sms-one-time-code-format/README.md

```diff
@@ -77,6 +89,16 @@ origin on which this code may be used.
 
 [schemelessly same site]: https://html.spec.whatwg.org/multipage/origin.html#schemelessly-same-site
 
+Adoption of this format would improve the reliability of systems which
+today heuristically extract one-time codes from SMS, with clear end-user
+benefit. It improves reliability of both extracting the code and also
+associating that code with an origin.
+
+Adoption of this proposal could improve the number of services on which
+a browser can offer SMS one-time code autofill, and could reduce the
```

Comment:
I would say, “improve the number of services on which a browser can offer assistance with providing SMS one-time codes to websites (e.g. an AutoFill feature)”.

Reply:
Addressed in 4a932ca.